assets/src/edit-story/components/canvas/gridview/index.js

```diff
@@ -58,13 +58,31 @@ const RangeInputWrapper = styled.div`
   margin: 0 auto 75px auto;
 `;
 
-const Rectangle = styled.div`
+const FlexGrowRangeInput = styled(RangeInput)`
+  flex-grow: 1;
+`;
+
+const Rectangle = styled.button`
+  border: 0;
+  padding: 0;
+  margin: 0;
+  min-width: unset;
+  background: transparent;
+  height: auto;
   display: flex;
   justify-content: center;
   align-items: center;
-  font-size: 32px;
+  cursor: pointer;
   color: ${({ theme }) => theme.colors.fg.v1};
-  ${({ isLarge }) => (isLarge ? 'margin-left' : 'margin-right')}: 20px;
+
+  &:active {
+    outline: none;
+  }
+
+  :disabled {
```

Comment:
Doesn't this require `&:disabled` to work?

Reply:
hmm... strange but it was working 😮

Reply:
Added  `&:disabled` 😄